### ISSUE TITLE
Unable to set priority group on pool member

### ISSUE THREAD

**Issue by chihaiaalex** [F5Networks/terraform-provider-bigip#381]
### Environment

 * TMOS/Bigip Version: BIG-IP 12.1.2 Build 2.0.276 Hotfix HF2
 * Terraform Version: v0.13.5
 * Terraform bigip provider Version: 1.4.0

### Summary

I'm trying to atach multiple nodes to a pool with different priority groups but it seems that the `priority_group` parameter is ignored.

### Steps To Reproduce

```
resource "bigip_ltm_pool" "example_pool" {
  name = "/k8s/k8s_example_pool"
  load_balancing_mode = "round-robin"
  description = "Example pool"
  monitors = [
    "/Common/tcp"
  ]
  allow_snat = "yes"
  allow_nat = "yes"
  minimum_active_members = 1
}

resource "bigip_ltm_pool_attachment" "example_pool_member_1" {
  pool = bigip_ltm_pool.example_pool.name
  node = "/k8s/example-node-1:31380"
  priority_group = 2
}

resource "bigip_ltm_pool_attachment" "example_pool_member_2" {
  pool = bigip_ltm_pool.example_pool.name
  node = "/k8s/example-node-2:31380"
  priority_group = 1
}
```

### Expected Behavior
First node is added to the pool with priority 2 and second node with priority 1.

### Actual Behavior
Both nodes are added to the pool with priority 0.

**Comment by nmenant** [comment]
We've been able to reproduce the issue. Tracking Internally as TER-511

**Comment by papineni87** [comment]
@chihaiaalex  I just pushed priority_group fix to master branch, can you check on it.

Below is the config, i tried


resource "bigip_ltm_node" "node1" {
  name        = "/test/node1.com"
  description = "Terraform managed"
  address     = "2.2.2.2"
}

resource "bigip_ltm_node" "node2" {
  name        = "/test/node2.com"
  description = "Terraform managed"
  address     = "3.3.3.3"
}

resource "bigip_ltm_pool" "example_pool" {
  name = "/test/k8s_example_pool"
  load_balancing_mode = "round-robin"
  description = "Example pool"
  allow_snat = "yes"
  allow_nat = "yes"
  minimum_active_members = 1
}

resource "bigip_ltm_pool_attachment" "example_pool_member_1" {
  pool = bigip_ltm_pool.example_pool.name
  node = "${bigip_ltm_node.node1.name}:80"
  priority_group = 2
}

resource "bigip_ltm_pool_attachment" "example_pool_member_2" {
  pool = bigip_ltm_pool.example_pool.name
  node = "${bigip_ltm_node.node2.name}:80"
  priority_group = 3
}

![image](https://user-images.githubusercontent.com/48274565/101236246-a2bb3f80-36f5-11eb-84fb-2eecd305659c.png)

**Comment by chihaiaalex** [comment]
@papineni87 Hi, sorry for the late response. I managed to test the new version and I see the problem is fixed when you add a new member with a priority group, but it doesn't work when you try to upgrade it, by changing its priority group.

**Comment by papineni87** [comment]
@chihaiaalex  Is it upgrade ? or update ? I just tried updating priority group and it worked. If there is an issue with upgrade, from which release to other release ? 

Below is the snapshot for update of priority group

$ terraform apply
bigip_ltm_node.node2: Refreshing state... [id=/test/node2.com]
bigip_ltm_pool.example_pool: Refreshing state... [id=/test/k8s_example_pool]
bigip_ltm_node.node1: Refreshing state... [id=/test/node1.com]
bigip_ltm_pool_attachment.example_pool_member_2: Refreshing state... [id=/test/k8s_example_pool-/test/node2.com:80]
bigip_ltm_pool_attachment.example_pool_member_1: Refreshing state... [id=/test/k8s_example_pool-/test/node1.com:80]

An execution plan has been generated and is shown below.
Resource actions are indicated with the following symbols:
  ~ update in-place

Terraform will perform the following actions:

  # bigip_ltm_pool_attachment.example_pool_member_1 will be updated in-place
  ~ resource "bigip_ltm_pool_attachment" "example_pool_member_1" {
        id             = "/test/k8s_example_pool-/test/node1.com:80"
        node           = "/test/node1.com:80"
        pool           = "/test/k8s_example_pool"
      ~ priority_group = 2 -> 4
    }

Plan: 0 to add, 1 to change, 0 to destroy.

Do you want to perform these actions?
  Terraform will perform the actions described above.
  Only 'yes' will be accepted to approve.

  Enter a value: yes

bigip_ltm_pool_attachment.example_pool_member_1: Modifying... [id=/test/k8s_example_pool-/test/node1.com:80]
bigip_ltm_pool_attachment.example_pool_member_1: Modifications complete after 1s [id=/test/k8s_example_pool-/test/node1.com:80]

Apply complete! Resources: 0 added, 1 changed, 0 destroyed.

**Comment by chihaiaalex** [comment]
@papineni87 is working now, I'm no longer able to reproduce it. I'm not sure why the first time it didn't work. I was trying to update a member by changing its priority group.

**Comment by papineni87** [comment]
Issue Addressed in v1.6.0, please open new issues if you face any problem
